app/src/main/java/com/moviefiend/torian/moviefiend/MovieListFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.app.Fragment;
+import android.app.LoaderManager;
+import android.content.Loader;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.moviefiend.torian.moviefiend.network.NowPlayingResponse;
+
+import java.util.ArrayList;
+
+public class MovieListFragment extends Fragment implements LoaderManager.LoaderCallbacks<ArrayList<NowPlayingResponse.MovieInfo>> {
+    public static final int MOVIE_LOADER_ID = 0;
+
+    private RecyclerView mRecyclerView;
+    private MovieAdapter mMovieAdapter;
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        getLoaderManager().initLoader(MOVIE_LOADER_ID, null, this);
```

Comment:
I'm not 100% sure but I think that if you call `initLoader` from `onActivityCreated` the `onLoadFinished` method will be called twice.